src/DynamoCore/Models/RunSettings.cs

```diff
@@ -20,8 +20,10 @@ public class RunSettings : NotificationObject
         private RunType runType;
         private bool runEnabled;
         private bool runTypesEnabled;
-        private bool runTypesComboBoxToolTipIsEnabled;
 
+        /// <summary>
+        /// Boolean indicates if Dynamo should block run even mode is Automatic.
```

Comment:
Should be updated to: `Boolean indicates if Dynamo should block run even when mode is Automatic.`

Reply:
Will do